arch/rv32i/src/lib.rs

```diff
@@ -227,7 +227,7 @@ pub extern "C" fn _start_trap() {
             // Jump to board-specific trap handler code. Likely this was an
             // interrupt and we want to disable a particular interrupt, but each
             // board/chip can customize this as needed.
-            jal ra, _start_trap_rust
+            jal ra, _start_trap_rust_from_kernel
```

Comment:
Why rename this?

Reply:
I didn't, this is from one of the commits I pulled recently to update my PR.

Reply:
Maybe it's best to rebase your patches then? The GitHub UI is showing me a large amount of other changes that it's hard to keep track of.

Reply:
But aren't the changed files easy to review the latest commit changes 🤔 ?

Reply:
I'm not clear on what you mean. It's much more difficult to follow changes with merge commits in the history. GitHub shows me a large amount of unrelated changes. A rebase on master is much cleaner to review and makes the Git history tidier and more bisectable.